interface/src/app/sample/sampleMocks.spec.js

```diff
@@ -0,0 +1,97 @@
+angular.module('adage.mocks.sample', [])
```

Comment:
So this file defines a module that is for test only? Since it doesn't include any test cases, it probably shouldn't be named "*.spec.js". 

I noticed that this module is needed by `sample_annotation/annotation.spec.js`. This makes me a little uneasy. I would prefer having test cases to be in separate units without this kind of extra dependencies. and have the tests in `sample_annotation/annotation.spec.js` provide fake data directly.